src/ExportableMesh.cpp

```diff
@@ -121,16 +121,32 @@ ExportableMesh::ExportableMesh(
 
                 ++vertexBufferIndex;
             }
-
+            GLTF::Object* morphTargetNames = new GLTF::MorphTargetNames;

```

Comment:
Replace with something else?

Reply:
In the version of `COLLADA2GLTF` that I'm using, the GLTF objects are not automatically deleted when destructing the GLTF root object. So to avoid memory leaks, you will have to add a data member to store the `MorphTargetNames`, e.g. in the header file, add `std::unique_ptr<GLTF::MorphTargetNames> m_morphTargetNames;`, and in the source file, use `m_morphTargetNames = std::make_unique<GLTF::MorphTargetNames>`.

Reply:
Added it to the class initialization. Hope that's ok.